# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: layout/tables/crashtests/1795051.html
# Commit: de9e12276d8e
# Full Hash: de9e12276d8eac6610b71706a1b2c5035b924e5b
# Author: Ting-Yu Lin <tlin@mozilla.com>
# Date: 2022-10-14 21:55:00
# Description:
#   Bug 1795051 - Fix the condition before calling ResetColIndices in nsTableFrame::InsertColGroups. r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D159354
# ==============================================================================

diff -r 8afa6eaecd2f -r de9e12276d8e layout/tables/crashtests/1795051.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/layout/tables/crashtests/1795051.html	Fri Oct 14 16:31:47 2022 +0000
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <style>
+        * {
+            max-width: fit-content;
+        }
+    </style>
+    <script>
+      window.addEventListener('load', () => {
+        const col = document.getElementById('id_0')
+        const table = document.getElementById('id_1')
+        const colgroup_1 = document.createElement('colgroup')
+        const colgroup_2 = document.createElement('colgroup')
+        colgroup_2.span = 174
+        table.appendChild(colgroup_1)
+        table.appendChild(col)
+        table.appendChild(colgroup_2)
+        document.documentElement.getAnimations({ })
+        colgroup_1.setAttribute('span', 190)
+        colgroup_2.setAttribute('span', 0)
+      })
+    </script>
+</head>
+<table id='id_1'>
+    <col id='id_0'>
+</table>
+</html>